src/core/lib/iomgr/exec_ctx.cc

```diff
@@ -25,10 +25,46 @@
 #include "src/core/lib/iomgr/combiner.h"
 #include "src/core/lib/profiling/timers.h"
 
-bool grpc_exec_ctx_ready_to_finish(grpc_exec_ctx* exec_ctx) {
+thread_local ExecCtx* exec_ctx = nullptr;
+
+ExecCtx::ExecCtx()
+    : closure_list(GRPC_CLOSURE_LIST_INIT),
```

Comment:
Prefer member initializers to constructor initializers to avoid duplication... that is:

```c++
class ExecCtx {
  /* ... */
  const unsigned starting_cpu_ = gpr_cpu_current_cpu();
};
```